chalicelib_fourfront/checks/wrangler_checks.py

```diff
@@ -1059,32 +1059,39 @@ def validate_entrez_geneids(connection, **kwargs):
         check.description = "Could not retrieve gene records from fourfront"
         return check
     geneids = [g.get('geneid') for g in genes]
-
-    query = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=gene&id={id}"
-    for gid in geneids:
+    ID_BATCH_SIZE = 150
+    query = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=gene&id={id}&rettype=uilist&retmode=text"
```

Comment:
I am not very familiar with this check, so I was testing the queries to see the response. I may be misunderstanding or not testing correctly, but it seems like this query just gives me back whatever numbers I put in my test ID, with no other info, regardless of whether it produces an error for the original query. Do you also see this behavior?

Reply:
That is the intended behavior I believe.  The ids are passed in as a comma separated list. Have you tried to include non-valid geneids?  Invalid IDs should not be included in the result list and then that list is checked against the input list and anything in input list not in the output is reported as problematic.  At least that is the idea.

Reply:
Yes, I mean that any string of digits will appear in the list, even invalid.

Reply:
Ah right you are!  Good catch.  If you put in something that is not a number you get something useful but it returns even invalid geneids as numbers.  OK I will need to fix that.